src/odfesqlodbc/es_parse_result.cpp

```diff
@@ -207,20 +230,22 @@ bool _CC_Metadata_from_ESResult(QResultClass *q_res, ConnectionClass *conn,
 }
 
 bool _CC_from_ESResult(QResultClass *q_res, ConnectionClass *conn,
-                       const char *cursor, ESResult &es_result) {
+                       const char *cursor, ESResult &es_result,
+                       schema_type* doc_schema) {
```

Comment:
Does doc_schema need to be passed as a pointer, it would be better as a reference especially as it's dereferenced here without checking against null.

Reply:
removed doc_schema